graphix/sim/statevec.py

```diff
@@ -335,15 +367,24 @@ def expectation_single(self, op, loc):
     def expectation_value(self, op, qargs):
         """Expectation value of multi-qubit operator.
 
-        Args:
-            op (np.array): 2^n*2^n operator
-            qargs (list of ints): target qubit
+        Parameters
+        ----------
+        op : numpy.ndarray
+            2^n*2^n operator
+        qargs : list of int
+            target qubit indices
 
-        Returns:
-            complex: expectation value
+        Returns
+        -------
+        complex: expectation value
         """
         st1 = deepcopy(self)
         st1.normalize()
         st2 = deepcopy(st1)
         st1.evolve(op, qargs)
         return np.dot(st2.psi.flatten().conjugate(), st1.psi.flatten())
+
+
+def get_norm(psi):
```

Comment:
why not add as a method (eg `_get_norm` if you don't want to explicitly expose it

Reply:
`_get_statevec_norm` sounds preferable to me!

Wildcard import is not recommended (https://peps.python.org/pep-0008/#imports) so we should avoid that.